packages/react-client/src/ReactFlightClientContext.js

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+const {useContext, useMemo, createContext} = React;
+
+const ServerContextContext = createContext(null);
```

Comment:
Thought provoking, but could this itself be a ServerContext? 🤯

If we wanted to enable this same hook to work from inside Server Components so that you can serialize the context for later use by other server components (like pre-rendering other subtrees).